tests/l2/qat/qatlib_job.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: intel-qat-workload
+  namespace: intel-qat
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: intel-qat-job
+        image: image-registry.openshift-image-registry.svc:5000/intel-qat/intel-qat-workload:latest
+        imagePullPolicy: IfNotPresent
+        command: ["./cpa_sample_code"]
+        securityContext:
+          capabilities:
+            add:
+              [IPC_LOCK]
+        resources:
+          requests:
+            qat.intel.com/dc: '1'
```

Comment:
Have we tried the case when we have more than one qat resource?

Reply:
yes. /dev/vfio will have 4 dev files instead of 2. since 2 of dc and 2 of cy resources are requested

Reply:
I mean have we ever tried to use multiple resources/VFs to do the dc or/and cy simultaneously in a single pod/container?

Reply:
yes difference is what /dev/vfio would have. we changed this job example, 2 cy and 2 dc.